src/agent/src/main.c

```diff
@@ -118,7 +118,7 @@ typedef void (*PnPComponentDestroyFunc)(void** componentContext);
  * @param result Result to report (optional, can be NULL).
  */
 typedef void (*PnPComponentPropertyUpdateCallback)(
-    IOTHUB_DEVICE_CLIENT_LL_HANDLE deviceClient,
+    ADUC_ClientHandle deviceClient,
```

Comment:
>deviceClient [](start = 22, length = 12)

here in this file also, if you decided to rename the 'deviceClient' param. (not a blocker)